packages/gestalt/src/Modal.js

```diff
@@ -104,18 +108,18 @@ const ModalWithForwardRef: React$AbstractComponent<Props, HTMLDivElement> = forw
     setShowBottomShadow(
       hasVerticalScrollbar && target.offsetHeight + target.scrollTop < target.scrollHeight,
     );
-  };
+  }, []);
```

Comment:
Do we want this callback to update if the target changes? (I _think_ the answer is "yes", but I'm interested in your thoughts)

Reply:
Short answer is: it's not allowed! We have the deep dependencies eslint in gestalt and no deps are needed. target is a useRef 
![image](https://user-images.githubusercontent.com/10593890/107578181-67d7fb00-6bc1-11eb-8361-011d23519369.png)

Reply:
good to know!